app/src/js/components/Collections/index.js

```diff
@@ -17,42 +16,91 @@ import DatePickerHeader from '../../components/DatePickerHeader/DatePickerHeader
 import { listCollections } from '../../actions';
 
 class Collections extends React.Component {
-  constructor () {
-    super();
-    this.displayName = strings.collection;
-  }
-
-  query () {
+  query() {
     this.props.dispatch(listCollections());
   }
 
-  render () {
+  render() {
     const { pathname } = this.props.location;
     const existingCollection = pathname !== '/collections/add';
 
     return (
-      <div className='page__collections'>
-        <DatePickerHeader onChange={this.query} heading={strings.collections}/>
-        <div className='page__content'>
-          <div className='wrapper__sidebar'>
-            <Route path='/collections/all' component={Sidebar} />
-            <Route path='/collections/edit/:name/:version' component={Sidebar} />
-            <Route path='/collections/collection/:name/:version' component={Sidebar} />
-            <div className={existingCollection ? 'page__content--shortened' : 'page__content'}>
+      <div className="page__collections">
```

Comment:
this diff is all format.  the rest of the file is mixed.  I pulled out withQueryParams as it wasn't used.